Source/ProgressionSystemRuntime/ProgressionSystemRuntime.Build.cs

```diff
@@ -11,10 +11,10 @@ public ProgressionSystemRuntime(ReadOnlyTargetRules Target) : base(Target)
 		PCHUsage = PCHUsageMode.UseExplicitOrSharedPCHs;
 		PublicDependencyModuleNames.AddRange(new[]
 			{
-				"Core", "UMG"
+				"Core", "UMG" 
+				, "GameFeatures"
```

Comment:
You don't need this "GameFeatures" dependency, since you don't use Observer anymore

Reply:
done with: https://github.com/h4rdmol/ProgressionSystem/pull/26/commits/09bfccd83e624edaaea9d505ebdd5e754500b050